leapp/cli/report/__init__.py

```diff
@@ -0,0 +1,79 @@
+"""
+This module is currently single workflow specific (upgrades). After introducing other use cases we can make it
+more robust
+"""
+
+import os
+
+from leapp.exceptions import CommandError
+from leapp.utils.clicmd import command, command_opt, command_arg
+from leapp.utils.report import fetch_upgrade_report_messages
+from leapp.cli.upgrade import fetch_last_upgrade_context
+
+from jinja2 import Template
+import json
+
+
+@command('report', help='Create report for last upgrade execution (default) or specific execution id using --id ID')
+@command_opt('id')
+@command_opt('--format')
```

Comment:
commands are a bit inconsistent, use --id

Reply:
We actually thought about having the `id` as positional argument. Makes most sense IMHO.

Reply:
@shaded-enmity @Rezney 
positional arguments use  @command_arg not @command_opt, just FYI